build.gradle

```diff
@@ -19,7 +22,7 @@ repositories {
 
 dependencies {
     implementation "dev.arbjerg.lavalink:plugin-api:0.7.0"
-    runtimeOnly "com.github.freyacodes.lavalink:Lavalink-Server:feature~plugins-SNAPSHOT"
+    runtimeOnly "com.github.freyacodes.lavalink:Lavalink-Server:feature~plugins"
```

Comment:
Why are you changing this?

Reply:
for me it couldn't resolve it with `-SNAPSHOT`
looking on jitpack it should hm
weird

Reply:
getting this with `-SNAPSHOT`
> Could not find Lavalink-Server-feature~plugins-SNAPSHOT.jar (com.github.freyacodes.lavalink:Lavalink-Server:feature~plugins-SNAPSHOT:3.0-gf5b2a17-574).